include/aca_vlan_manager.h

```diff
@@ -56,19 +55,16 @@ class ACA_Vlan_Manager {
 
   int delete_ovs_port(string vpc_id, string ovs_port, uint tunnel_id, ulong &culminative_time);
 
-  int create_neighbor_outport(string neighbor_id, string vpc_id,
-                              alcor::schema::NetworkType network_type, string remote_host_ip,
-                              uint tunnel_id, ulong &culminative_time);
+  int create_l2_neighbor(string virtual_ip, string virtual_mac, string remote_host_ip,
```

Comment:
Not sure the best practice for C/C++ header file. But do we have a place to write down comments to describe those important functions, interfaces in the header files and beyond?

Reply:
For example, we actually don't know what the response integer stands for.

Reply:
yes, I am thinking about the adopt Doxygen commenting, what do you think?

https://devblogs.microsoft.com/cppblog/doxygen-and-xml-doc-comment-support/
https://devblogs.microsoft.com/cppblog/visual-studio-code-c-extension-july-2020-update-doxygen-comments-and-logpoints/

Reply:
This is a very good tool and coding practice. Let us plan to add it throughput the ACA repo and ask other engineers to follow the coding practice as well.